src/index.html

```diff
@@ -11,8 +11,54 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
SCSS files should not be linked directly in HTML. Use the compiled CSS file instead.